msg/json/en.json

```diff
@@ -1,370 +1,9 @@
-	"LISTS_GET_SUBLIST_END_FROM_START": "to #",
-	"LISTS_GET_SUBLIST_END_FROM_END": "to # from end",
-	"LISTS_GET_SUBLIST_END_LAST": "to last",
-	"LISTS_GET_SUBLIST_TAIL": "",
-	"LISTS_GET_SUBLIST_TOOLTIP": "Creates a copy of the specified portion of a list.",
-	"LISTS_SPLIT_HELPURL": "https://github.com/google/blockly/wiki/Lists#splitting-strings-and-joining-lists",
-	"LISTS_SPLIT_LIST_FROM_TEXT": "make list from text",
-	"LISTS_SPLIT_TEXT_FROM_LIST": "make text from list",
-	"LISTS_SPLIT_WITH_DELIMITER": "with delimiter",
-	"LISTS_SPLIT_TOOLTIP_SPLIT": "Split text into a list of texts, breaking at each delimiter.",
-	"LISTS_SPLIT_TOOLTIP_JOIN": "Join a list of texts into one text, separated by a delimiter.",
-	"ORDINAL_NUMBER_SUFFIX": "",
-	"VARIABLES_GET_HELPURL": "https://github.com/google/blockly/wiki/Variables#get",
-	"VARIABLES_GET_TITLE": "",
-	"VARIABLES_GET_TAIL": "",
-	"VARIABLES_GET_TOOLTIP": "Returns the value of this variable.",
-	"VARIABLES_GET_CREATE_SET": "Create 'set %1'",
-	"VARIABLES_SET_HELPURL": "https://github.com/google/blockly/wiki/Variables#set",
-	"VARIABLES_SET_TITLE": "set",
-	"VARIABLES_SET_TAIL": "to",
-	"VARIABLES_SET_TOOLTIP": "Sets this variable to be equal to the input.",
-	"VARIABLES_SET_CREATE_GET": "Create 'get %1'",
-	"PROCEDURES_DEFNORETURN_HELPURL": "https://en.wikipedia.org/wiki/Procedure_%28computer_science%29",
-	"PROCEDURES_DEFNORETURN_TITLE": "to",
-	"PROCEDURES_DEFNORETURN_PROCEDURE": "do something",
-	"PROCEDURES_BEFORE_PARAMS": "with:",
-	"PROCEDURES_CALL_BEFORE_PARAMS": "with:",
-	"PROCEDURES_DEFNORETURN_DO": "",
-	"PROCEDURES_DEFNORETURN_TOOLTIP": "Creates a function with no output.",
-	"PROCEDURES_DEFRETURN_HELPURL": "https://en.wikipedia.org/wiki/Procedure_%28computer_science%29",
-	"PROCEDURES_DEFRETURN_RETURN": "return",
-	"PROCEDURES_DEFRETURN_TOOLTIP": "Creates a function with an output.",
-	"PROCEDURES_ALLOW_STATEMENTS": "allow statements",
-	"PROCEDURES_DEF_DUPLICATE_WARNING": "Warning: This function has duplicate parameters.",
-	"PROCEDURES_CALLNORETURN_HELPURL": "https://en.wikipedia.org/wiki/Procedure_%28computer_science%29",
-	"PROCEDURES_CALLNORETURN_CALL": "",
-	"PROCEDURES_CALLNORETURN_TOOLTIP": "Run the user-defined function '%1'.",
-	"PROCEDURES_CALLRETURN_HELPURL": "https://en.wikipedia.org/wiki/Procedure_%28computer_science%29",
-	"PROCEDURES_CALLRETURN_TOOLTIP": "Run the user-defined function '%1' and use its output.",
-	"PROCEDURES_MUTATORCONTAINER_TITLE": "inputs",
-	"PROCEDURES_MUTATORCONTAINER_TOOLTIP": "Add, remove, or reorder inputs to this function.",
-	"PROCEDURES_MUTATORARG_TITLE": "input name:",
-	"PROCEDURES_MUTATORARG_TOOLTIP": "Add an input to the function.",
-	"PROCEDURES_HIGHLIGHT_DEF": "Highlight function definition",
-	"PROCEDURES_CREATE_DO": "Create '%1'",
-	"PROCEDURES_IFRETURN_TOOLTIP": "If a value is true, then return a second value.",
-	"PROCEDURES_IFRETURN_WARNING": "Warning: This block may be used only within a function definition."
+
```

Comment:
It appears to be an issue with /msg/json/en.js having the trailing comma. Removing it made the compile happy.